Formula/idris2.rb

```diff
@@ -12,19 +13,19 @@ class Idris2 < Formula
     sha256 cellar: :any, mojave:   "847646c54f71dea739dfb24e167c41d8a14c9c02e1f6d08fa639d5a468c6d4cf"
   end
 
+  depends_on "coreutils" => :build
   depends_on "gmp" => :build
   depends_on "chezscheme"
-  depends_on "coreutils"
   uses_from_macos "zsh" => :build, since: :mojave
 
   def install
     ENV.deparallelize
     scheme = Formula["chezscheme"].bin/"chez"
     system "make", "bootstrap", "SCHEME=#{scheme}", "PREFIX=#{libexec}"
     system "make", "install", "PREFIX=#{libexec}"
-    # idris2.so is an executable file generated by Idris2
-    bin.install "#{libexec}/bin/idris2_app/idris2.so" => "idris2"
```

Comment:
This is no longer needed because `bin/idris2` _runner_ now uses `zsh` on macOS:
```sh
#!/bin/sh

set -e # exit on any error

if [ "$(uname)" = Darwin ]; then
  DIR=$(zsh -c 'printf %s "$0:A:h"' "$0")
else
  DIR=$(dirname "$(readlink -f -- "$0")")
fi
export LD_LIBRARY_PATH="$DIR/idris2_app":$LD_LIBRARY_PATH
"$DIR/idris2_app/idris2.so" "$@"
```